package.json

```diff
@@ -1,21 +1,33 @@
 {
-  "name": "sumo-ot-js",
+  "name": "sumo-opentelemetry",
   "version": "0.0.0",
-  "private": true,
+  "description": "Sumo Logic all-in-one library for OpenTelemetry tracing auto-instrumentation in web",
+  "main": "./dist/src/index.js",
```

Comment:
Do we need to have the `src` directory in `dist`? Looks a bit weird.

Reply:
Can be directly in `dist`, but I cannot configure where the `index.d.ts` file will be placed, so I've decided to leave everything there.

Reply:
Why not set `rootDir` in tsconfig to `src` if that's where the whole source lies?

Reply:
Because I use git submodules located in `/opentelemetry-js` and `/opentelemetry-js-contrib`. Also there is a `lib` directory where I put our fork of Jaeger exporter (`opentelemetry-exporter-sumologic`). I'm thinking about moving this exporter to `src`, still not sure about that.

Reply:
I see. I'd move all of them to `src`, but this is just a suggestion, nice paths are not critical in any way here.

Reply:
lib and git modules now are in `src/`.